client/my-sites/marketing/do-it-for-me/difm-lite-in-progress.tsx

```diff
@@ -35,6 +43,22 @@ function DIFMLiteInProgress( { siteId }: DIFMLiteInProgressProps ) {
 	const translate = useTranslate();
 	const dispatch = useDispatch();
 
+	const sitePurchases = useSelector( ( state ) => getSitePurchases( state, siteId ) );
```

Comment:
The computation from the line 46 to 60 here will all go into void if `isWebsiteContentSubmitted` is truthy. Could we move them below to where they will definitely be needed?

Reply:
Good call out! This was a sign that the two different states of this component (content submitted/unsubmitted) were diverging from each other more, so I have separated them into separate components. This should make it easier to read.

Reply:
That's so much cleaner. Thanks for going extra miles, making it even better than I would have expected :)